README_CN.md

```diff
@@ -84,551 +84,283 @@ Qwen2.5-Omni在包括图像，音频，音视频等各种模态下的表现都
-</details>
+|--------------------------|-----------------|---------------|----------------|----------------|
+| Refcoco<sub>val</sub>    | 90.5            | 90.0          | **90.6**       | 73.2           | 
+| Refcoco<sub>textA</sub>  | **93.5**        | 92.5          | 93.2           | 72.9           | 
+| Refcoco<sub>textB</sub>  | 86.6            | 85.4          | **88.2**       | 74.6           | 
+| Refcoco+<sub>val</sub>   | 85.4            | 84.2          | **88.2**       | 62.5           | 
+| Refcoco+<sub>textA</sub> | **91.0**        | 89.1          | 89.0           | 63.9           | 
+| Refcoco+<sub>textB</sub> | **79.3**        | 76.9          | 75.9           | 65.0           | 
+| Refcocog+<sub>val</sub>  | **87.4**        | 87.2          | 86.1           | 75.2           | 
+| Refcocog+<sub>test</sub> | **87.9**        | 87.2          | 87.0           | 76.2           | 
+| ODinW                    | 42.4            | 37.3          | **55.0**       | 36.7           | 
+| PointGrounding           | 66.5            | **67.3**      | -              | -              | 
 
+</details>
 
 <details>
 <summary>Video(without audio) -> Text</summary>
 
 | Dataset                     | Qwen2.5-Omni-7B | Other Best | Qwen2.5-VL-7B | GPT-4o-mini | 
-|-----------------------------|--------------|------------|---------------|-------------|
-| Video-MME<sub>w/o sub</sub> | 64.3         | 63.9       | **65.1**      | 64.8        | 
-| Video-MME<sub>w sub</sub>   | **72.4**     | 67.9       | 71.6          | -           | 
-| MVBench                     | **70.3**     | 67.2       | 69.6          | -           | 
-| EgoSchema<sub>test</sub>    | **68.6**     | 63.2       | 65.0          | -           | 
+|-----------------------------|-----------------|------------|---------------|-------------|
+| Video-MME<sub>w/o sub</sub> | 64.3            | 63.9       | **65.1**      | 64.8        | 
+| Video-MME<sub>w sub</sub>   | **72.4**        | 67.9       | 71.6          | -           | 
+| MVBench                     | **70.3**        | 67.2       | 69.6          | -           | 
+| EgoSchema<sub>test</sub>    | **68.6**        | 63.2       | 65.0          | -           | 
+
 </details>
 
 <details>
 <summary>Zero-shot Speech Generation</summary>
 
+#### Content Consistency
+**Dataset:** SEED
+
+| Model              | test-zh | test-en | test-hard |
+|--------------------|---------|---------|-----------|
+| Seed-TTS_ICL       | 1.11    | 2.24    | 7.58      |
+| Seed-TTS_RL        | **1.00**| 1.94    | **6.42**  |
+| MaskGCT            | 2.27    | 2.62    | 10.27     |
+| E2_TTS             | 1.97    | 2.19    | -         |
+| F5-TTS             | 1.56    | **1.83**| 8.67      |
+| CosyVoice 2        | 1.45    | 2.57    | 6.83      |
+| CosyVoice 2-S      | 1.45    | 2.38    | 8.08      |
+| Qwen2.5-Omni-7B_ICL | 1.70    | 2.72    | 7.97      |
+| Qwen2.5-Omni-7B_RL  | 1.42    | 2.32    | 6.54      |
```

Comment:
```suggestion
| Qwen2.5-Omni-7B_RL  | 1.42    | 2.33    | 6.54      |
```
<img width="496" alt="Screenshot 2025-04-08 at 23 48 43" src="https://github.com/user-attachments/assets/08b7591a-0c76-42e4-849d-1b0f1b28c9e8" />
Typo in original markdown.